app/views/welcome/placeholder.html.erb

```diff
@@ -3,15 +3,43 @@
   <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track' => true %>
 </head>
 <body class="placeholder">
+  <%= render "application/video" %>
+
   <div class="valign-wrapper bg-wrapper">
     <div class="valign center-align center z-depth-3 bg-ribbon">
       <%= image_tag("coposition.png", style: "margin: 2em") %>
-      <h1 class="brand" data-in-effect="bounceIn">CONTROL YOUR LOCATION DATA</h1>
+      <h1 class="brand" data-in-effect="bounceIn">coposition</h1>
       <h5>Worry free location data management.</h5>
-  <script type="text/javascript">
-         window.Maitre = { uuid: "MFfa39989fd3" };
-  </script>
-  <script data-maitre src='https://maitreapp.co/widget.js' async></script>
+      <p>
+
+      <script type="text/javascript">
+             window.Maitre = { uuid: "MFfa39989fd3" };
+      </script>
+      <script data-maitre src='https://maitreapp.co/widget.js' async></script>
+      <script type="text/javascript">
+        responsiveVideo();
+
+        function responsiveVideo() {
+          var ratio = 1920/1080;
+          // var h = $(".promo").height();
+          // var w = $(".promo").width();
+          var h = document.getElementsByClassName("promo")[0].clientHeight;
+          var w = document.getElementsByClassName("promo")[0].clientWidth;
+          var rRatio = w/h;
+
+          if(rRatio < ratio) {
+            // Aspect ratio is lower than 16:9
+            document.getElementsByTagName("video")[0].style.width = "auto";
+            document.getElementsByTagName("video")[0].style.height = "100%";
+          }else{
+            // Aspect ratio is higher than 16:9
+            document.getElementsByTagName("video")[0].style.width = "100%";
+            document.getElementsByTagName("video")[0].style.height = "auto";
+          }
+        }
+
```

Comment:
Hey can you take me through what's happening here? I notice this is similar to responsiveVideo in init.js.